.mergify.yml

```diff
@@ -0,0 +1,65 @@
+pull_request_rules:
+  - name: delete head branch after merge
+    conditions: []
+    actions:
+      delete_head_branch: {}
+  - name: automatic merge for softwaremill-ci pull requests affecting build.sbt
+    conditions:
+      - author=softwaremill-ci
+      - check-success=ci
+      - "#files=1"
+      - files=build.sbt
+    actions:
+      merge:
+        method: merge
+  - name: automatic merge for softwaremill-ci pull requests affecting project plugins.sbt
+    conditions:
+      - author=softwaremill-ci
+      - check-success=ci
+      - "#files=1"
+      - files=project/plugins.sbt
+    actions:
+      merge:
+        method: merge
+  - name: semi-automatic merge for softwaremill-ci pull requests
+    conditions:
+      - author=softwaremill-ci
+      - check-success=ci
+      - "#approved-reviews-by>=1"
+    actions:
+      merge:
+        method: merge
+  - name: automatic merge for softwaremill-ci pull requests affecting project build.properties
+    conditions:
+      - author=softwaremill-ci
+      - check-success=ci
+      - "#files=1"
+      - files=project/build.properties
+    actions:
+      merge:
+        method: merge
+  - name: automatic merge for softwaremill-ci pull requests affecting .scalafmt.conf
+    conditions:
+      - author=softwaremill-ci
+      - check-success=ci
+      - "#files=1"
+      - files=.scalafmt.conf
+    actions:
+      merge:
+        method: merge
+  - name: automatic merge for softwaremill-ci pull requests affecting project Versions.scala
+    conditions:
+      - author=softwaremill-ci
+      - check-success=ci
+      - "#files=1"
+      - files=project/Versions.scala
```

Comment:
**question:** there is no `project/Versions.scala` in the project in question - maybe it should be added as a part of this PR otherwise this rule seems to be useless 😄